packages/backend-data/src/convert_authorization_modes.ts

```diff
@@ -150,6 +150,20 @@ const computeApiKeyAuthFromResource = (
   };
 };
 
+const authorizationModeMapping = {
```

Comment:
Could you type this as a Record<DefaultAuthorizationMode, CDKDefaultAuthorizationMode>?

Reply:
I had some of this in an earlier version. My making the mapping `const` we get this strictness for free without any additional public exporting. Would recommend we leave it and add this on a refactor.